enclave_manager/avalon_enclave_manager/graphene/graphene_enclave_info.py

```diff
@@ -25,19 +28,70 @@ class SignupGraphene():
     """
 
 # -------------------------------------------------------------------------
-    def __init__(self, worker_signup_json):
+    def __init__(self, config, worker_signup_json):
         """
         Constructor for SignupGraphene.
         Creates signup object.
         """
+        self._connect_to_ias(config['EnclaveModule'])
```

Comment:
Can we call this function from `_generate_avr()`. We need to contact IAS only if quote is not empty

Reply:
Done. Verified after rebase.